src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -764,6 +776,8 @@ cli_drop_node_getopts(int argc, char **argv)
 	static struct option long_options[] = {
 		{ "pgdata", required_argument, NULL, 'D' },
 		{ "destroy", no_argument, NULL, 'd' },
+		{ "nodename", required_argument, NULL, 'n' },
+		{ "nodeport", required_argument, NULL, 'p' },
```

Comment:
we used --pgport when creating nodes. We should stick to that naming scheme.

Reply:
Good idea, done that way now.